src/compat/qsort.c

```diff
@@ -255,3 +258,10 @@ _quicksort (void *const pbase, size_t total_elems, size_t size,
   }
 }
 #endif /* !HAVE_QSORT_R_GLIBC */
+#else
+#if _WIN32
```

Comment:
If the intention is to target the VS compiler then the macro to use must be _MSC_VER. _WIN32 affects also the MinGW compiler (gcc with I don't know whats mixes)

Reply:
OK, fixed